resources/views/contact-auth.blade.php

```diff
@@ -58,6 +58,9 @@ class="py-3 px-4 block w-full transition ease-in-out duration-150" />
                         @enderror
                     </div>
 
+                    {{-- honeypot to detect bots filling out the form --}}
+                    <input type="hidden" name="age" />
+
```

Comment:
I'll check this PR more thoroughly next week, but I don't think bots would fill out a hidden input. I'd look up "honeypot" but IIRC you at least want an actual input, just visually hidden via some (ideally not too obvious) CSS.